src/main/java/step3/domain/Car.java

```diff
@@ -6,14 +6,26 @@
 public class Car {
     private static final int INIT_DISTANCE = 0;
     public static final String NO_ROUND_EXCEPTION_MESSAGE = "해당하는 라운드의 경기기록이 없습니다.";
+    public static final String NAME_LENGTH_EXCEPTION_MESSAGE = "자동차의 이름은 5자를 초과할 수 없습니다.";
+    private static final int NAME_LENGTH_LIMIT = 5;
 
+    private String name;
```

Comment:
```suggestion
    private final String name;
```

현재의 요구사항에서 이름은 변하지 않아도 될 것 같아요, 위와 같이 선언하면 어떨까요?

Reply:
매번 잊어먹는 요소들 중 하나인 것 같습니다. 잘 생각해야겠습니다. 감사합니다.